apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -207,6 +207,22 @@ function formatVote(vote: ApiVote): Vote {
   };
 }
 
+async function formatUser(user: User) {
```

Comment:
In previous PR I recommended we remove this helper because it was used only once and the types there were not accurate as well.

Right now the same issue happens, but looking more deeply into it it looks like it has more issues, maybe we can solve this in different PR if you want. We should address issue with the types (`user` isn't of `User` type, it has two cases:
1. Only `{ id }`
2. Only `{ id, name, about, avatar, cover, github, twitter, lens, farcaster }`, missing are `created` (we probably should query that), `follows`, `proposal_count`, `vote_count`

I'd even say those last 3 shouldn't be part of `User` as it will always be zero, and Leaderboard should have it's own type, because in turn Leaderboard has same `User` type, but missing are other fields (it has counts, id and name but is missing everything else).

Reply:
We currently this

```
export type UserProfile = {
  name?: string;
  about?: string;
  avatar?: string;
  cover?: string;
  github?: string;
  twitter?: string;
  lens?: string;
  farcaster?: string;
};

export type User = {
  id: string;
  proposal_count: number;
  vote_count: number;
  created?: number;
  follows?: string[];
  name?: string;
} & UserProfile;
```

Maybe we could use `UserProfile` only for everything return by `loadUser` (maybe rename this as well by `loadUserProfile`), and keep `User` for everything else (leaderboard, etc...).

This will also require rename usersStore to userProfilesStore, the User view to UserProfile, and all routes to user-profile

This will remove all confusion about those 2 entities.

`proposal_count` and `vote_count` should be removed from `User` and `UserProfile`, this stats will always be computed from the leaderboard, by summing the count on all networks.

Reply:
> Maybe we could use UserProfile only for everything return by loadUser (maybe rename this as well by loadUserProfile), and keep User for everything else (leaderboard, etc...).

Is there anything else? It looks like we use it in two places (user profile page and leaderboard).
To avoid issues with renaming stores/views I'd just go with approach with keeping `User` named as-is for the sake of User profile page, and for leaderboard use `LeaderboardUser`/`LeaderboardEntry`/`LeaderboardItem` or whatever you think makes most sense there.